src/pytype.jl

```diff
@@ -292,7 +292,7 @@ end
 
 # Often, PyTypeObject instances are global constants, which we initialize
 # to 0 via PyTypeObject() and then initialize at runtime via PyTypeObject!
-function PyTypeObject!(t::PyTypeObject, name::AbstractString, basicsize::Integer, init::Function)
+function PyTypeObject!(init::Function, t::PyTypeObject, name::AbstractString, basicsize::Integer)
```

Comment:
Note that I moved the `init` function to go first here because that allows us to use `do` syntax.

This is an undocumented internal function so I'm guessing we should be okay without adding a deprecation?    I can't find any packages using this low-level code to define their own `pyjlwrap` subtypes.